sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/polling/PollOperation.java

```diff
@@ -7,142 +7,28 @@
 import com.azure.core.http.HttpPipeline;
 import com.azure.core.http.HttpRequest;
 import com.azure.core.http.HttpResponse;
-import com.azure.core.http.rest.Response;
 import com.azure.core.management.polling.PollResult;
-import com.azure.core.util.FluxUtil;
 import com.azure.core.util.logging.ClientLogger;
 import com.azure.core.util.polling.LongRunningOperationStatus;
 import com.azure.core.util.polling.PollResponse;
-import com.azure.core.util.polling.PollerFlux;
 import com.azure.core.util.polling.PollingContext;
 import com.azure.core.util.serializer.SerializerAdapter;
 import com.azure.core.util.serializer.SerializerEncoding;
-import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 import java.io.IOException;
 import java.lang.reflect.Type;
-import java.nio.ByteBuffer;
-import java.nio.charset.StandardCharsets;
-import java.time.Duration;
-import java.util.Objects;
 import java.util.function.BiFunction;
 import java.util.function.Function;
 
 /**
- * Factory to create PollerFlux for Azure resource manager (ARM) long-running-operation (LRO).
+ * Poll operation for Azure resource manager (ARM) long-running-operation (LRO).
  */
-public final class PollerFactory {
-    private static final ClientLogger LOGGER = new ClientLogger(PollerFactory.class);
+public final class PollOperation {
```

Comment:
The remaining public method (used by `polling.PollerFactory`) in the class is for pollingOperation, cancelOperation and fetchResultOperation, hence changing the class name to `PollOperation`.

Let me know if any suggestion.